public-api/db/changelog/db.changelog-master.xml

```diff
@@ -9,7 +9,8 @@
     <runningAs username="liquibase"/>
   </preConditions>
   <include file="changelog/db.changelog-1.xml"/>
+  <include file="changelog/db.changelog-2.xml"/>
```

Comment:
The environments all already exist so it's probably too late to be changing the migration files around, aside from renaming files. I suppose we could delete the databases if we really wanted to... Did you have a plan for this?